bigquery_etl/metadata/parse_metadata.py

```diff
@@ -230,9 +231,13 @@ def from_file(cls, metadata_file):
         with open(metadata_file, "r") as yaml_stream:
             try:
                 metadata = yaml.safe_load(yaml_stream)
-
-                friendly_name = metadata.get("friendly_name", None)
-                description = metadata.get("description", None)
+                table_name = str(metadata_file.parent.name)
+                friendly_name = metadata.get(
+                    "friendly_name", string.capwords(table_name.replace("_", " "))
+                )
+                description = metadata.get(
+                    "description", "Please provide a description."
```

Comment:
nit: matching the `query create` [default description](https://github.com/mozilla/bigquery-etl/blob/ac0af012c2654546c76bf11105c7c6cb06b9c33e/bigquery_etl/cli/query.py#L224): 
```suggestion
                    "description", "Please provide a description for the query",
```
We should probably consolidate this logic somehow.